<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceRIOMDetailPage</class>
 <widget class="QWidget" name="MaintainceRIOMDetailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 128);
	border-radius:8px;
	border-top: 2px solid white;	
	border-left: 2px solid white;
}</string>
  </property>
  <widget class="QPushButton" name="TCDI1Btn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>20</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TC-DI1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TCDI2Btn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TC-DI2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TCDI3Btn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TC-DI3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TCDI4Btn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>170</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TC-DI4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TCDI5Btn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>220</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TC-DI5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TCAIMBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>320</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TC-AIM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MPDIBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>370</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>MP-DI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MDIBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>470</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>M-DI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TCDOBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>270</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TC-DO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MPDOBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>420</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>MP-DO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MDOBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>520</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>M-DO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>610</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>TCDI1Btn</zorder>
  <zorder>TCDI2Btn</zorder>
  <zorder>TCDI3Btn</zorder>
  <zorder>TCDI4Btn</zorder>
  <zorder>TCDI5Btn</zorder>
  <zorder>TCAIMBtn</zorder>
  <zorder>MPDIBtn</zorder>
  <zorder>MDIBtn</zorder>
  <zorder>TCDOBtn</zorder>
  <zorder>MPDOBtn</zorder>
  <zorder>MDOBtn</zorder>
  <zorder>returnBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
